utils/fixture-helper/src/resolver.ts

```diff
@@ -0,0 +1,50 @@
+import * as globby from 'globby';
+import Path = require('path');
+
+interface ResolutionsByTestDir {
+  [testDir: string]: Record<string, string>;
+}
+
+interface ResolverOptions {
+  basedir: string;
+  defaultResolver: (request: string, options: ResolverOptions) => string;
+  extensions: string[];
+  moduleDirectory: string[];
+  paths: string[];
+  rootDir: string[];
+}
+
+function testDirFromResolutionPath(resPath: string): string {
+  return Path.normalize(Path.parse(resPath).dir);
+}
+function testDirFromBase(baseDir: string): string {
+  return Path.join(baseDir, '../__tests__');
+}
+
+let mapping: ResolutionsByTestDir = {};
+
+globby
+  .sync([`**/__tests__/module-resolution.json`], {
+    cwd: process.cwd(),
+    absolute: true,
+    ignore: ['node_modules', '.yarn', '.github', 'docs', 'private']
+  })
+  .forEach(file => {
+    const testDir = testDirFromResolutionPath(file);
+    mapping[testDir] = require(file);
+  });
+
+function resolver(path: string, options: ResolverOptions): string {
+  const testDir = testDirFromBase(options.basedir);
+
+  let actualResolution: string | null = null;
+
+  if (mapping[testDir]) {
+    const resolutionInFile = mapping[testDir][path];
+    actualResolution = resolutionInFile && Path.join(testDir, resolutionInFile);
+  }
+
+  return actualResolution || options.defaultResolver(path, options);
+}
+
+module.exports = resolver;
```

Comment:
and export too :)

Reply:
It's pretty weird, but when I change to 

`export default resolver;`

In the compiled file, I have:

`exports.default = resolver;`

as expected, but when I run the tests I then get:

![image](https://user-images.githubusercontent.com/34415120/89231266-e3d30400-d5dc-11ea-8b4a-5735ab4f083a.png)

It's as if the resolver option of jest.config does not support `exports.default` syntax.

Reply:
oh I forgot it's used by jest directly, probably they only support commonjs, then it's ok you can leave it